Lib/test/test_buffer.py

```diff
@@ -978,7 +978,7 @@ def match(req, flag):
             lst = nd.tolist()
 
         # The consumer may have requested default values or a NULL format.
-        ro = 0 if match(req, PyBUF_WRITABLE) else ex.readonly
+        ro = not match(req, PyBUF_WRITABLE) and ex.readonly
```

Comment:
Could you please use the previous version, just replacing `0` with `False`?

I can accept `not p or q` for implication, but this goes too far IMHO.